<launch>
  <node name="velocity_smoother" pkg="robot_control" type="velocity_smoother_node" output="screen">
    <rosparam file="$(find robot_control)/param/velocity_smoother.yaml" command="load"/>
    <remap from="velocity_smoother/raw_cmd_vel"    to="/twist_cmd_filtered"/>
    <remap from="velocity_smoother/smooth_cmd_vel" to="/twist_controller/command"/>
  </node>
</launch>